src/ks/hangedman/Colour.java

```diff
@@ -0,0 +1,23 @@
+package ks.hangedman;
+
+public enum Colour {
+
+    RED("\u001B[31m"),
+    GREEN ("\u001B[32m"),
+    BLACK ("\u001B[30m"),
+    YELLOW ("\u001B[33m"),
+    BLUE ("\u001B[34m"),
+    PURPLE ("\u001B[35m"),
+    CYAN ("\u001B[36m"),
+    RESET ("\u001B[0m");
```

Comment:
Should you allow to use RESET color to be passed as argument to the method? It seems confusing. I would use this ```"\u001B[0m"``` constant only directly in ```colourLetters(...)``` method and do not put in as enum value here.